src/i18n/pl.json

```diff
@@ -0,0 +1,145 @@
+{
+  "comboBox": {
```

Comment:
all component keys should be placed in the parent key “components”.  This will bring more order to this file. There will be no situation that we have a component keys at the beginning and at the end of the file. This file should resemble the structure of the project's directory tree as much as possible